.github/workflows/pr-test.yml

```diff
@@ -37,16 +37,50 @@ jobs:
 
     # job level env vars used for a couple of steps
     env:
-      # see github-glue.test.ts for info on setting these
       GGG_TOKEN: ${{ secrets.GGG_TOKEN }}
       GGG_REPOSITORY: ${{ secrets.GGG_REPOSITORY }}
+      GGG_SMTP_USER: ${{ secrets.GGG_SMTP_USER }}
+      GGG_SMTP_PASS: ${{ secrets.GGG_SMTP_PASS }}
+      GGG_SMTP_HOST: ${{ secrets.GGG_SMTP_HOST }}
 
     steps:
     # Check out repo under sub-dir so other repo can be checked out
     - uses: actions/checkout@v2
       with:
         path: gitgitgadget
 
+    - name: Set git test repo config
+      shell: bash
+      run: |
+        if [ -n "$GGG_TOKEN" ] && [ -n "$GGG_REPOSITORY" ];
+        then
+          echo "::set-env name=GGG_REPO::'gitgitgadget.githubtest.githubuser=$GITHUB_REPOSITORY_OWNER' 'gitgitgadget.$GITHUB_REPOSITORY_OWNER.githubrepo=$GGG_REPOSITORY' 'gitgitgadget.$GITHUB_REPOSITORY_OWNER.githubtoken=$GGG_TOKEN'"
+        else
+          echo "::error::Both GGG_TOKEN and GGG_REPOSITORY secrets must be set"
+          exit 1
```

Comment:
I guess it does not hurt much to have an `exit 1` here, but I would have expected the `echo ::error::...` to mark this task as failed, and consequently all subsequent tasks to be skipped anyway.